lib/Service/OpenProjectAPIService.php

```diff
@@ -491,4 +496,66 @@ public static function validateOpenProjectURL(string $openprojectUrl): bool {
 		return filter_var($openprojectUrl, FILTER_VALIDATE_URL) &&
 			preg_match('/^https?/', $openprojectUrl);
 	}
+
+	/**
+	 * authenticated request to get status of a work package
+	 *
+	 * @param string $url
+	 * @param string $accessToken
+	 * @param string $authType
+	 * @param string $refreshToken
+	 * @param string $clientID
+	 * @param string $clientSecret
+	 * @param string $userId
+	 * @param string $statusId
+	 * @return string[]
+	 * @throws NotFoundException
+	 * @throws NotPermittedException
+	 * @throws LockedException
+	 * @throws Exception
+	 */
+	public function getOpenProjectWorkPackageStatus(string $url, string $accessToken, string $authType, string $refreshToken, string $clientID,
+													string $clientSecret, string $userId, string $statusId): array
+	{
```

Comment:
formatting suggestion

```suggestion
	public function getOpenProjectWorkPackageStatus(
		string $url,
		string $accessToken,
		string $authType,
		string $refreshToken,
		string $clientID,
		string $clientSecret,
		string $userId,
		string $statusId
	): array
	{
```

also let's not use `$authType` in new functions, it will be deleted anyway soon. Where-ever you need to pass it give `oauth`